tests/lib/prepare.sh

```diff
@@ -25,6 +25,21 @@ disable_kernel_rate_limiting() {
     #sysctl -w kernel.printk_ratelimit=0
 }
 
+disable_journald_rate_limiting() {
+    # Disable journald rate limiting
+    mkdir -p /etc/systemd/journald.conf.d
```

Comment:
This fails on core16 because /etc/systemd/journal.conf.d is not a writable location. However, on core18, the whole /etc/systemd is writable.

Reply:
https://github.com/snapcore/core/pull/109 <- should fix this.